app/models/fe/concerns/choice_field_concern.rb

```diff
@@ -126,13 +126,21 @@ def display_response(app=nil)
     def conditional_match(answer_sheet)
       displayed_response = display_response(answer_sheet)
       (is_true(displayed_response) && is_true(conditional_answer)) ||
-        (is_false(displayed_response) && is_false(conditional_answer)) ||
+        (is_response_false(answer_sheet) && is_false(conditional_answer)) ||
         (displayed_response == conditional_answer)
     end
 
+    def is_response_true(answer_sheet)
```

Comment:
I added it to be symmetrical with is_response_false.. I can take it out if you prefer.